task/task_smoke_test.go

```diff
@@ -42,7 +74,17 @@ func TestTaskSmoke(t *testing.T) {
 	if testName == "" {
 		testName = "smoke test"
 	}
-
+	script := `
+#!/bin/sh -e
+if [ -n "$TEST_GPU" ]; then
+  nvidia-smi
+fi
+mkdir --parents cache output
+touch cache/file
+echo "$ENVIRONMENT_VARIABLE_DATA" | tee --append output/file
+sleep 60
+cat output/file
+`[1:]
```

Comment:
My eyes! Can we leave this string where it was? 😅 

I'd also mildly prefer a slightly shorter version like e.g. `test -n "$TEST_GPU" && nvidia-smi` or `printenv TEST_GPU && nvidia-smi` instead of a conditional spanning three lines. 💅🏼

Reply:
Do we have a newline quota for the codebase?

Reply:
No, we don't, although we don't get paid by the number of lines we write either. 😂 

There is a [natural tendency for compactness](https://github.com/iterative/terraform-provider-iterative/pull/443#pullrequestreview-910911048) in the team you may follow or not. In terms of code duplication there are stronger preferences, but things like this are up to you.

Reply:
What about `if test -v TEST_GPU; then nvidia-smi; fi` in a single line? 😼🧶

Reply:
Moved to https://github.com/iterative/terraform-provider-iterative/pull/673/files#r974235102 & https://github.com/iterative/terraform-provider-iterative/pull/673/files#r974237957. Thanks!